definitions/npm/big.js_v3.x.x/test_big.js_v3.x.x.js

```diff
@@ -1,6 +1,5 @@
 // @flow
-
-import Big from "big.js";
+import Big from 'big.js';
```

Comment:
I changed it to keep things consistent but I get the point. Thanks for explaining :)